base/linalg.jl

```diff
@@ -212,6 +212,7 @@ include("linalg/exceptions.jl")
 include("linalg/generic.jl")
 
 include("linalg/blas.jl")
+import .BLAS: gemv! # consider renaming gemv! in matmul
```

Comment:
It looks ugly to have an `import` here, so I was reminded that, in general, I think we should just call the multiplication methods `mul!` when we have the transpose type. The `ge` and `v` information is carried in the argument types. The name `m!` is a bit short though :smile:.